doctest/doctest.h

```diff
@@ -1313,6 +1329,11 @@ DOCTEST_CLANG_SUPPRESS_WARNING_POP
         int         m_expected_failures;
         double      m_timeout;
 
+        // Fix segfault on Intel Compiler 2021 during the DOCTEST_TEST_SUITE_END execution
```

Comment:
Weird - I can't think of why it would segfault and there are no issues with all the other compilers & sanitizers & valgrind... Any clue why this might be happening?

Reply:
I have rechecked without it. I guess it was a workaround for older versions of Intel Compiler 2021. It is not needed.